dockermodelrunner/desktop/desktop.go

```diff
@@ -0,0 +1,508 @@
+package desktop
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"html"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
suggestion: remove use of pkg/errors it predates stdlib wrapable errors and so is no longer required bloating imports